gp-includes/misc.php

```diff
@@ -325,3 +325,40 @@ function gp_gmt_strtotime( $string ) {
 
 	return $time;
 }
+
+function gp_personal_options( $user ) {
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+	
+?>
+<tr>
+<th scope="row"><?php _e( 'GlotPress Administrator', 'glotpress' ); ?></th>
+<td><fieldset><legend class="screen-reader-text"><span><?php _e( 'GlotPress Administrator', 'glotpress' ); ?></span></legend>
+<label for="gp_administrator">
+<input name="gp_administrator" type="checkbox" id="gp_administrator" value="1"<?php checked( GP::$permission->user_can( $user, 'admin' ) ); ?> />
+<?php _e( 'If checked, this user will be a GlotPress administrator', 'glotpress' ); ?></label><br />
+</fieldset>
+</td>
+</tr>
+
+<?php
+}
+
+function gp_personal_options_update( $user_id ) {
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+	
+	$current_gp_admin = GP::$permission->user_can( $user_id, 'admin' );
```

Comment:
`$current_gp_admin` is confusing (it's not the current user), `$is_user_gp_admin` makes more sense IMO.